### DISCUSSION TITLE
PTZ camera suggestions

### DISCUSSION THREAD

**Discussion by darlezior** [blakeblackshear/frigate#9938] [General]
Good morning everyone, could you recommend some models of PTZ cameras that work with frigates (even in autotracking)? I did a bit of research but I couldn't find references to particular models to be able to make a purchase. Thank you very much for any useful advice.

**Comment by hawkeye217** [comment]
This was just asked the other day [here](https://github.com/blakeblackshear/frigate/discussions/9903). 

The [docs](https://docs.frigate.video/configuration/cameras/#onvif-ptz-camera-recommendations) have a user-submitted, growing list of supported cameras. In general, Dahua and Amcrest cameras seem to have the best support for autotracking. For basic PTZ controls, any ONVIF-capable PTZ is likely to be supported, even if not listed in the docs.
